{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/templates/{{ cookiecutter.project_name }}/home.jinja

```diff
@@ -35,6 +35,9 @@
       <h1>mozilla webdev</h1>
     </header>
     It works! Now build something.
+    {% raw %}
+      {% csrf_token %}
+    {% endraw %}
```

Comment:
This is exactly the case here. This will be first parsed by cookiecutter which also uses Jinja template engine and we want to split out `{% csrf_token %}`